tests/Console/ConfigurationResolverTest.php

```diff
@@ -954,6 +954,123 @@ public function testResolveRulesWithOption(): void
         );
     }
 
+    /**
+     * Test the renamed rules
+     * 
+     * @dataProvider provideRenamedRulesCases
+     * 
+     * @return void
+     */
+    public function testResolveRenamedRulesWithUnknownRules(string $expectedMessage, string $renamedRule): void
+    {
+        $this->expectException(
+            \PhpCsFixer\ConfigurationException\InvalidConfigurationException::class
+        );
+        $this->expectExceptionMessage($expectedMessage);
+
+        $resolver = $this->createConfigurationResolver(['rules' => $renamedRule]);
+
+        $resolver->getRules();
+    }
+    
+    public function provideRenamedRulesCases()
+    {
+        return [
+            [
+                'The rules contain unknown fixers: "blank_line_before_return" is a renamed rule,  did you mean "blank_line_before_statement"? note: use configuration [\'statements\' => [\'return\']]
```

Comment:
```suggestion
                'The rules contain unknown fixers: "blank_line_before_return" is a renamed rule,  did you mean "blank_line_before_statement" (note: use configuration "[\'statements\' => [\'return\']]")?
```

Reply:
Done in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5887/commits/f292fcfc5c5f30bbb99d465bde5410a632f7b6f9